configuration_schemas/v4/VERSION.json.in

```diff
@@ -1,5 +1,5 @@
 {
-    "id": "file:///var/lib/irods/configuration_schemas/v4/VERSION.json",
+    "$id": "file:///@IRODS_HOME_DIRECTORY@/configuration_schemas/v4/VERSION.json",
```

Comment:
@SwooshyCueb Is this enough for non-package installs? Should `CMAKE_INSTALL_PREFIX` be used here as well?

I noticed that it is used with `IRODS_HOME_DIRECTORY` in server/CMakeLists.txt (see https://github.com/irods/irods/pull/6275/files#r835943065) to update the schema files.

I know that `file:////` appeared when I tried using it and the validator didn't have any issues. I'm guessing that is because `////` is collapsed into `///`.

Reply:
The `file:///` results in an absolute path rather than a relative path.

For people not familiar with this, `file:///` is equivalent to `file://localhost/`.

See https://en.wikipedia.org/wiki/File_URI_scheme for more details.

Reply:
Honestly I think having absolute paths written down anywhere is problematic. Not just for non-package installs, but also for relocatable packages and tarball packages.

Reply:
I'll create an issue about absolute paths and we can go on and merge this.

Reply:
Done.